docs/src/_includes/components/rule-categories.macro.html

```diff
@@ -32,7 +32,8 @@
     <div class="rule-category">
         <span class="rule-category__icon">❄️ <span class="visually-hidden">Frozen</span></span>
         <p class="rule-category__description">
-            This rule is currently frozen and is not accepting changes.
+            This rule is currently frozen and is not accepting <a
+                href="https://github.com/eslint/eslint/#stylistic-rule-updates">feature requests</a>
```

Comment:
What do you think about this solution? I think this will allow people who want to learn more to find information about it faster. Since it is not at all obvious that you need to look for it in the README

Reply:
Great! We can include a link. Let's get other people's opinions too.

Reply:
The link seems useful to me, but how about also updating the description of frozen rules in `Readme` as it only mentions stylistic rules as frozen?

Reply:
I have changed the title in README per https://github.com/eslint/eslint/issues/19346#issuecomment-2598878884.

What is the best way to change the description? Should I completely remove the mention of stylistic rules and just leave general information about frozen rules?